OWNERS

```diff
@@ -0,0 +1,15 @@
+# The OWNERS file is used by prow to automatically merge approved PRs.
+
+approvers:
+- bbrowning
+- greghaynes
+- jcrossley3
+- kleung
+- evankanderson  # TOC member as backup
+- mattmoor       # TOC member as backup
+
+reviewers:
```

Comment:
What do reviewers mean?

Reply:
You are correct that anyone can be a reviewer.
The reviewer section is used by the bots to auto-assign/suggest reviewers. 

This is to ensure that the TOC members that are backup approvers are not spammed by review requests.

If you like, I can add you you to the reviewer section.

Reply:
I definitely want to be one.

Reply:
added

Reply:
@k4leung I also would like to be reviewer.  Could you please add me to the list?

Reply:
@yu2003w added.